### PR TITLE
Updated Akka to 1.5.1

### PR DESCRIPTION
**Pull request by eaba**
## Changes

- Update `Akka.NET` to 1..5.1
- Update `Quartz.NET` to 3.6.2
- Update `Microsoft.NET.Test.Sdk` to 17.5.0

## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).
* [x] I have [reviewed my own pull request](https://getakka.net/community/contributing/index.html#review-your-own-pull-requests).

### CODE REVIEW

File: src/Akka.Quartz.Actor/QuartzActor.cs [hunk 1]
```diff
@@ -65,7 +65,17 @@ public QuartzActor(IScheduler scheduler)
 
         protected override bool Receive(object message)
         {
-            return message.Match().With<CreateJob>(CreateJobCommand).With<RemoveJob>(RemoveJobCommand).WasHandled;
+            switch (message)
```
Comment (eaba) [inline src/Akka.Quartz.Actor/QuartzActor.cs:68]: changed it into switch @Arkatufus

File: src/Akka.Quartz.Actor/QuartzPersistentActor.cs [hunk 2]
```diff
@@ -38,7 +38,17 @@ protected override void OnSchedulerCreated(IScheduler scheduler)
 
         protected override bool Receive(object message)
         {
-            return message.Match().With<CreatePersistentJob>(CreateJobCommand).With<RemoveJob>(RemoveJobCommand).WasHandled;
+            switch (message)
```
Comment (eaba) [inline src/Akka.Quartz.Actor/QuartzPersistentActor.cs:41]: changed it into switch @Arkatufus

File: src/common.props [hunk 3]
```diff
@@ -2,9 +2,9 @@
   <PropertyGroup>
     <Copyright>Copyright © 2015-2021 .NET Foundation</Copyright>
     <Authors>Akka</Authors>
-    <VersionPrefix>1.4.19</VersionPrefix>
-    <PackageReleaseNotes>Updated Akka to 1.4.19
-upgraded to Quartz.NET 3.3.2</PackageReleaseNotes>
+    <VersionPrefix>1.5.1</VersionPrefix>
```
Comment (eaba) [inline src/common.props:5]: Updated Akka.NET to 1.5.1

File: src/common.props [hunk 4]
```diff
@@ -13,10 +13,10 @@ upgraded to Quartz.NET 3.3.2</PackageReleaseNotes>
     </NoWarn>
   </PropertyGroup>
   <PropertyGroup>
-    <AkkaVersion>1.4.49</AkkaVersion>
-    <QuartzVersion>3.6.0</QuartzVersion>
+    <AkkaVersion>1.5.1</AkkaVersion>
+    <QuartzVersion>3.6.2</QuartzVersion>
```
Comment (eaba) [inline src/common.props:17]: Updated Quartz.NET to 3.6.2

File: src/common.props [hunk 5]
```diff
@@ -13,10 +13,10 @@ upgraded to Quartz.NET 3.3.2</PackageReleaseNotes>
...
     <XunitVersion>2.4.2</XunitVersion>
     <XunitRunnerVersion>2.4.5</XunitRunnerVersion>
-    <TestSdkVersion>17.4.1</TestSdkVersion>
+    <TestSdkVersion>17.5.0</TestSdkVersion>
```
Comment (eaba) [inline src/common.props:20]: Updated `Microsoft.NET.Test.Sdk` to 17.5.0

**Review by eaba — COMMENTED** [review]
Updated Akka.NET to 1.5.1 @Arkatufus

File: src/Akka.Quartz.Actor/QuartzActor.cs (hunk 1 above)
Comment (Aaronontheweb) [reply src/Akka.Quartz.Actor/QuartzActor.cs:68]: LGTM

File: src/Akka.Quartz.Actor/QuartzPersistentActor.cs (hunk 2 above)
Comment (Aaronontheweb) [reply src/Akka.Quartz.Actor/QuartzPersistentActor.cs:41]: LGTM

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**